clang/test/Sema/attr-nonblocking-sema.cpp

```diff
@@ -1,6 +1,8 @@
 // RUN: %clang_cc1 -fsyntax-only -fblocks -fcxx-exceptions -verify %s
 // RUN: %clang_cc1 -fsyntax-only -fblocks -verify -x c -std=c23 %s
 
+#pragma clang diagnostic warning "-Wfunction-effects"
```

Comment:
Er, why is this here instead of in the `RUN` lines?

Reply:
Moved